src/components/home/elements/TokenRecords.vue

```diff
@@ -3,6 +3,9 @@
        class="records">
     <div class="title-records">
       <span>Token Watch List</span>
```

Comment:
We'd better change `Token Watch List` to `Token watch list`. It will give illusion that text is not higher than (+).